src/ConstructiveSolidGeometry/LinePrimitives/Edge.jl

```diff
@@ -0,0 +1,18 @@
+struct Edge{T} <: AbstractLinePrimitive{T}
+    a::CartesianPoint{T}
+    b::CartesianPoint{T}
+end
+
+direction(e::Edge) = e.b - e.a
```

Comment:
I would assume `direction` to be normalized to one. Is this somehow relevant?

Reply:
Well, `direction` is in general just a direction. You do not always need it normalized. 
And if so one can just wrap it in `normalize(direction(e))` which is not that long. And is becomes clearer as well. 
Same for the `direction` of a line. It's not normalized in the constructor.